_docs/_latest/use/using-instrumentation.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Instrumentation
+order: 200
+---
+
+# {{ page.title }}
+
+Opencensus tracing has been added to Forseti as part of instrumentation on 
+Forseti.
+
+---
+
+**Tracing**
+* Tracing is enabled by default and the traces are automatically exported to 
+`Stackdriver Trace`, and viewable in GCP console.
+* Traces are extremely helpful for latency analysis of gRPC calls. Viewing 
+traces in GCP console will keep you informed about the time taken in ms to 
+create an inventory and/or perform various operations on the model. This data 
+is critical in analysing and debugging latency issues. 
+* Stackdriver Exporter is the default exporter for traces. Feel free to add in 
+support for the exporter of your choice as OpenCensus supports variety of 
+exporters such as Zipkin, Jaeger and contribute back. 
+
```

Comment:
Should we add a `What's next` section, so that people can read more about OpenCensus.

![image](https://user-images.githubusercontent.com/6977544/47738989-a08e7900-dc31-11e8-9f20-78608789efa7.png)

Reply:
Future work will be decided based on needs and users' request.